src/app-toolkit/helpers/master-chef/master-chef.contract-position-helper.ts

```diff
@@ -268,7 +269,15 @@ export class MasterChefContractPositionHelper {
         }
 
         // Resolve data properties
-        const dataProps = { poolIndex, totalValueLocked, isActive, dailyROI, weeklyROI, yearlyROI };
+        const dataProps = {
+          poolIndex,
+          totalValueLocked,
+          liquidity: totalValueLocked,
```

Comment:
As a follow-up, are you planning on renaming `resolveTotalValueLocked` to `resolveLiquidity`? I think that'd make sense.

Reply:
yes, I am. I wanted to keep changes as smol as possible reducing potential errors